mantidproject/mantid
# Correctly identify display indices for non-orthog transform in sliceviewer

**Description of work.**

Small bug fix to correctly identify the display indices from slice info object when setting up transform for non-orthogonal view. This caused bug when switching viewing axes in non-orthog view.

**To test:**

See instructions in original issue, when  switching the viewing axes to a non-orthogonal pair in orthogonal view, the axes of the gridlines should orthogonal.

Fixes #30311



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
